### DISCUSSION TITLE
Postcss 8 error while having postcss 8

### DISCUSSION THREAD

**Discussion by Scraayp** [tailwindlabs/tailwindcss#5984] [Help] [answered]
**What version of Tailwind CSS are you using?**

^2.2.19
**What build tool (or framework if it abstracts the build tool) are you using?**

"postcss": "^8.3.11", "@vue/cli-service": "~4.5.0", "vue": "^3.0.0", "autoprefixer": "^10.4.0",

**What version of Node.js are you using?**

v16.13.0

**What browser are you using?**

Opera GX Browser | Using chrome

**What operating system are you using?**

Windows 11

**Reproduction URL**

I don't actually have a reproduction I do just link the source code of the full package for you:
https://github.com/Scraayp/bhcshop

**Describe your issue**

I'm getting this errror:
```
error  in ./src/assets/index.css

Syntax Error: Error: PostCSS plugin tailwindcss requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users


 @ ./src/assets/index.css 4:14-163 15:3-20:5 16:22-171
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8080&sockPath=/sockjs-node (webpack)/hot/dev-server.js ./src/main.js
```

While I deffinetly have postcss 8 installed like you could notice. I don't know what I' m doing wrong then.

**Comment by adamwathan** [comment, marked as answer]
Definitely have PostCSS 7 installed, many versions of it actually:

https://github.com/Scraayp/bhcshop/blob/main/yarn.lock#L5878-L5940

I think you need to trace down what dependencies you have installed that are bringing along PostCSS 7 and clean things up such that only PostCSS 8 is being installed. Hope that helps!

**Reply by Scraayp** [reply]
You have any suggestion how to find the dependencie which is doing this?

**Reply by Scraayp** [reply]
> You have any suggestion how to find the dependencie which is doing this?

Nevermind I have found this page, https://tailwindcss.com/docs/installation#post-css-7-compatibility-build
